samples/instrumentation-quickstart/src/index.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import axios from 'axios';
+import Fastify from 'fastify';
+import {loggerConfig, randInt, sleep} from './util';
+
+const port = 8080;
+
+// [START opentelemetry_instrumentation_create_app]
+// Create the Fastify app providing the Pino logger config
+const fastify = Fastify({
+  logger: loggerConfig,
+});
+// [END opentelemetry_instrumentation_create_app]
+
+// [START opentelemetry_instrumentation_handle_single]
+/**
+ * handleSingle handles an http request by sleeping for 100-200 ms. It writes the number of
+ * milliseconds slept as its response.
+ */
+fastify.get('/single', async request => {
+  // Sleep between 100-200 milliseconds
+  const sleepMillis = randInt(100, 200);
+  request.log.info({sleepMillis}, 'Going to sleep');
```

Comment:
Non-blocking: Neat syntax.  Does this end up structured?

Reply:
Yup!
![image](https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/assets/1510004/ce5c435f-96e6-49e4-b7f5-c8be6d48d9d0)